modules/spo-container-profile-instances.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-container-profile-instances_{context}"]
+= Merging per-container profile instances
+
+By default, each container instance records into a separate profile. The Security Profiles Operator can merge the per-container profiles into a single profile.
+
+.Procedure
+
+. Edit a `ProfileRecording` object to include `mergeStrategy: containers`:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileRecording
+metadata:
+  # The name of the Recording is the same as the resulting {kind} CRD
+  # after reconciliation.
+  name: test-recording
+spec:
+  kind: {kind}
+  recorder: logs
+  mergeStrategy: containers
+  podSelector:
+    matchLabels:
+      app: sp-record
+----
+
+. Create the workload:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deploy
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: sp-record
+  template:
+    metadata:
+      labels:
+        app: sp-record
+    spec:
+      serviceAccountName: spo-record-sa
+      containers:
+      - name: nginx-record
+        image: quay.io/security-profiles-operator/test-nginx-unprivileged:1.21
+        ports:
+        - containerPort: 8080
+----
+
+. To record the individual profiles, delete the deployment by running the following command:
++
+[source,terminal]
+----
+$ oc delete deployment nginx-deploy
+----
+
+. To merge the profiles, delete the profile recording by running the following command:
++
+[source,terminal]
+----
+$ oc delete profilerecording test-recording
+----
+
```

Comment:
This is really cool and I don't have any real comments, just one question for you:
once the deployment is deleted, profiles would be already created, just not installed. The profiles created based on a `ProfileRecording` with `mergeStrategy: containers` can only ever have status of `Partial`. Do you think this is too low level information or something that would be handy for the user to know in case they run `oc get selinuxprofiles` or `oc get seccompprofiles` before they delete the recording?

Reply:
I considered this. 

Because the `oc get` and `Partial` status does not necessarily contribute to the procedure I left it out. I couldn't think of a way to include that in a note either, or how a customer could benefit from knowing about the Partial status. But I understand where you're coming from, too. I'll continue thinking about it!

Reply:
(Sorry, I thought I already replied to this)

OK, I get your point and I agree. I like that the docs show the flow as the user would follow it and doesn't stray into details. The recording is complex as it is.

Thanks!